### DISCUSSION TITLE
Alwasys listens for "Ok Google"

### DISCUSSION THREAD

**Discussion by ghost** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#297] [Q&A]
Hi!
Is there any simple solution for enabling "always listens for "Ok Google"?
Just a quick fix, before the major patch?

**Comment by paddyboo999** [comment]
As far as i'm aware, i do not think that the assistant can be activated using a wake word. Sorry!

**Comment by sumeshir26** [comment]
@OxCone This has been implemented in the build version and will be released soon...
